scripts/make_toy_data.py

```diff
@@ -235,33 +218,38 @@ def fetch_cells(prefix):
     print('Generating matrix')
     letters = ['A', 'B', 'C', 'D']
 
-    bytes_per_column = 4.7 * num_rows  # ~1.65 KB (KiB) per 80 cells, uncompressed
+    # ~1.65 KB (KiB) per 80 cells, uncompressed
+    bytes_per_column = 4.7 * num_rows
+
     global num_columns
     if not num_columns:
         num_columns = int(bytes_per_file/bytes_per_column)
     # Generate header
     barcodes = []
     header = 'GENE\t'
-    # if we have a preloaded barcodes file, read it in, otherwise generate the random barcodes
+    # if we have a preloaded barcodes file, read it in, otherwise generate
+    # the random barcodes
     if preloaded_barcodes:
         with open(preloaded_barcodes) as f:
             # load preloaded barcodes/cell names
             lines = f.readlines()
             barcodes = [line.strip() for line in lines if len(line) > 2]
             if num_columns > len(barcodes):
-                # if user param --num_barcdes is higher than the number in the preloaded file, drop it down
+                # if user param --num-barcodes is higher than the number in the
+                # preloaded file, drop it down
                 print('Not enough barcodes in preloaded file, reducing gene number to', len(genes))
```

Comment:
The content of this print statement seems inaccurate
```suggestion
                print('Not enough barcodes in preloaded file, reducing barcode number to', len(barcodes))
```

Reply:
Nice catch!  Fixed in 5474edd.